.gitignore

```diff
@@ -14,3 +14,5 @@ web/sites/*/files
 
 # Ignore local settings
 web/sites/default/settings.local.php
+
+web/sites/
```

Comment:
There are files that we want to track in dis directory. Can you remove this line?